libs/translator/reader/read_geometry.cpp

```diff
@@ -668,6 +672,13 @@ void UsdArnoldReadCube::Read(const UsdPrim &prim, UsdArnoldReaderContext &contex
     for (GfVec3f& pt : points)
         pt = scale.Transform(pt);
 
+    VtValue sidednessValue;
```

Comment:
it looks like all the added code for setting the sidedness is the same for all the different type of prim, we could have a function `void SetSidedNess(UsdGeomGprim &)` ?

Reply:
absolutely. Done